app-backend/migrations/src_migrations/main/scala/54.scala

```diff
@@ -0,0 +1,15 @@
+import slick.driver.PostgresDriver.api._
+import com.liyaos.forklift.slick.SqlMigration
+
+object M54 {
+  RFMigrations.migrations = RFMigrations.migrations :+ SqlMigration(54)(List(
+    sqlu"""
+    CREATE TABLE organization_features (
+      organization UUID NOT NULL UNIQUE,
+      feature_flags UUID[],
```

Comment:
I assumed that features were presumed disabled unless flagged, so the flag ID itself indicated enabled (or vice versa) if tagged onto the record.